src/test/java/org/generationcp/middleware/service/pedigree/AncestryTreeServiceTest.java

```diff
@@ -149,11 +149,57 @@ private Germplasm generateTestGermplasm(final int gid, final int methodId) {
 		germplasm.setGid(gid);
 		germplasm.setGnpgs(2);
 		final Method method = new Method(methodId);
+		if(methodId > 1) {
+			method.setMname("Backcross");
+			germplasm.setGpid1(0);
```

Comment:
Maybe better to use non-zero as parent GIDs?

Reply:
Removed this unit test. Unit test are in PedigreeServiceImplTest